classes/PrestaShopAutoload.php

```diff
@@ -81,14 +81,24 @@ public static function getInstance()
         return static::$instance;
     }
 
+    /**
+     * Get cache directory.
+     *
+     * @return string
+     */
+    protected static function getCacheDir()
+    {
+        return dirname(__DIR__) . '/var/cache';
```

Comment:
Same comment than this one : https://github.com/PrestaShop/PrestaShop/pull/28121/files#r859984678

Reply:
yes, this PR depends on the #28121, let's merge it first